emsdk

```diff
@@ -26,15 +26,14 @@ if [ -z "$EMSDK_PYTHON" ]; then
     # This is needed for MacOS.  Without this, the urlopen
     # code will try to use /usr/local/etc/openssl/cert.pem
     # which may or may not exist on the system.
-    export SSL_CERT_FILE=$PYTHON3_CERT_FILE
+    export SSL_CERT_FILE="$PYTHON3_CERT_FILE"
   fi
 fi
 
 # If bundled python is not found, look for `python3` in PATH.  This is especially important on macOS (See:
 # https://github.com/emscripten-core/emsdk/pull/273)
 if [ -z "$EMSDK_PYTHON" ]; then
-  PYTHON3=$(which python3 2> /dev/null)
-  if [ $? = 0 ]; then
+  if PYTHON3="$(which python3 2>/dev/null)"; then
```

Comment:
Are you sure its safe to combine these two lines?

Reply:
Yes, it is expected to be safe across all POSIX compliant shells, but I did do some digging after you asked just to see if I could find any exceptions to this and was not able to. It seems consistent that combining the command with the check is the recommended approach. More juicy/useful info on the this at [ShellCheck: SC2181 – Check exit code directly with e.g. `if mycmd;`, not indirectly with `$?`.](https://www.shellcheck.net/wiki/SC2181). Thanks for reviewing this!